.readthedocs.yaml

```diff
@@ -7,9 +7,9 @@ version: 2
 
 # Set the version of Python and other tools
 build:
-  os: ubuntu-20.04
+  os: ubuntu-22.04
```

Comment:
Did you consider using `ubuntu-latest` instead? 

I'm afraid that we will need to update this in a few years again, no?

All in all, this doesn't prevent us from merging this PR.

Reply:
I did't consider it, but I prefer keeping the way it is. I my experience, when Ubuntu updates its version, there might be some instabilities with packages that are migrating or made breaking changes.

Updating the os is important, but it happens so infrequently that I believe doing it manually makes more sense for us.